shared/src/business/useCases/getMaintenanceModeInteractor.ts

```diff
@@ -7,8 +7,13 @@
 export const getMaintenanceModeInteractor = async (
   applicationContext: IApplicationContext,
 ) => {
+  const start = Date.now();
+  console.log('Start getMaintenanceModeInteractor');
```

Comment:
while I do wish to debug this endpoint, I don't think these `console.log` commands will be completely helpful. I may opt for `applicationContext.logger.info(...)`, but I think the X-Ray indicators will also be most informative.